api/app/src/routes/medical/organization.ts

```diff
@@ -20,22 +25,72 @@ router.post(
   asyncHandler(async (req: Request, res: Response) => {
     const cxId = getCxIdOrFail(req);
 
-    // TODO: parse this into model
-    const orgData = req.body;
+    const reqOrgData = organizationSchema.parse(req.body);
+
+    // TODO: I FIGURED THESE 3 WOULD BE GOOD TO STORE IN DB
+    // CAN ADD GATEWAYS OR TECHNICAL CONTACT ETC.
+    const localOrgPayload = {
+      name: reqOrgData.name,
+      type: "Hospital",
+      locations: reqOrgData.locations,
+    };
 
     // update if this is an existing org
-    let org: Organization;
-    if (orgData.id) {
-      const data = { ...orgData };
+    let localOrg: Organization;
+
+    if (reqOrgData.id) {
+      const data = { ...reqOrgData };
       delete data.id;
-      org = await updateOrganization({ id: orgData.id, cxId, data });
+      localOrg = await updateOrganization({ id: reqOrgData.id, cxId, data: localOrgPayload });
     } else {
-      org = await createOrganization({ cxId, data: orgData });
+      localOrg = await createOrganization({ cxId, data: localOrgPayload });
     }
 
-    // TODO: create or update organization in CW as well
+    const cwOrgPayload: CWOrganization = {
+      ...localOrgPayload,
+      // NOTE: IN STAGING IF THE ID ALREADY EXISTS IT WILL SAY INVALID ORG WHEN CREATING
+      organizationId: `${CW_ID_PREFIX}${localOrg.id}`,
+      homeCommunityId: `${CW_ID_PREFIX}${localOrg.id}`,
+      patientIdAssignAuthority: `${CW_ID_PREFIX}${localOrg.id}`,
+      displayName: reqOrgData.name,
+      memberName: "Metriport",
+      securityTokenKeyType: "BearerKey",
+      isActive: true,
+      gateways: [
+        {
+          serviceType: "XCA_Query",
+          gatewayType: "R4",
+          endpointLocation: Config.getGatewayEndpointLocation(),
+        },
+      ],
+      authorizationInformation: {
+        authorizationServerEndpoint: Config.getGatewayAuthorizationServerEndpoint(),
+        clientId: Config.getGatewayAuthorizationClientId(),
+        clientSecret: Config.getGatewayAuthorizationClientSecret(),
+        documentReferenceScope: "fhir/document",
+        binaryScope: "fhir/document",
+      },
+      technicalContacts: [
+        {
+          name: "Metriport Team",
+          title: "Engineering",
+          email: "support@metriport.com",
+          phone: "(415)-941-3282",
```

Comment:
We should externalize this at some point. Could at least ticket if if you prefer to leave it for later.